bark/models/behavior/behavior_rss/behavior_rss.cpp

```diff
@@ -75,6 +77,7 @@ Trajectory BehaviorRSSConformant::Plan(
   dynamic::Trajectory last_traj;
   if (behavior_rss_status_ == BehaviorRSSConformantStatus::NOMINAL_BEHAVIOR) {
     // execute normal
+    ApplyRestrictionsToNominalModel();
```

Comment:
It is somewhat hidden that we use a member variable here to set the restrictions. Better pass the restrictions to the function.